src/test/fuzz/CMakeLists.txt

```diff
@@ -56,7 +56,8 @@ add_executable(fuzz
   locale.cpp
   merkleblock.cpp
   message.cpp
-  miniscript.cpp
+  # TODO: Fix the code and enable miniscript.cpp for MSVC.
+  $<$<NOT:$<CXX_COMPILER_ID:MSVC>>:miniscript.cpp>
```

Comment:
What's necessary here?

Reply:
I don't know at this moment.

Otherwise, compiling with MSVC fails as reported in https://github.com/bitcoin/bitcoin/pull/29774#issuecomment-2028819335.

Reply:
Did you check if it is fixed by #28657 @hebasto?

Reply:
Checking now...